src/main/riddl/Venues/Locations/locationMessages.riddl

```diff
@@ -0,0 +1,58 @@
+command EstablishLocation is {
+  locationId: LocationId briefly "A unique identifier for this Location. This must be provided by the calling client.",
+  name: String,
+  description: String,
+  directions: String,
+  latLong: LatLong,
+  url: URL?,
+  capacity: Integer?,
+  accessible: Boolean,
+  amenities: String*,
+  entrances: Address+,
+  reservations: ReservationId*
+} briefly "Creates a Location with all top-level fields, potentially without Locations"
+
+event LocationEstablished is {
+  locationId: LocationId,
+  locationInfo: LocationContext.Info,
+  reservations: ReservationId*,
+  meta: LocationContext.MetaInfo
+}
+
+command AddLocationReservations is {
```

Comment:
I don't think we should have AddLocationReservation. I think this should be handled in the Reservation Context and you get Reservations for a Location by query against a Projection.